Index: src/com/data/controller/UserController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/com/data/controller/UserController.java b/src/com/data/controller/UserController.java
--- a/src/com/data/controller/UserController.java	(revision 39925328a1124cf8661b02cd88bbe9bb03560a79)
+++ b/src/com/data/controller/UserController.java	(revision ceb4f56627fe4b3755efa37c3bb13140423a62cc)
@@ -5,6 +5,7 @@
     //Needs to be impliment
     public String setData(){
         System.out.println("setData");
+        //test for a patch
         return null;
     }
 
